docker-compose.yaml

```diff
@@ -53,6 +58,37 @@ services:
     expose:
       - "50052"
 
+  opa-service:
```

Comment:
Can you name this just "opa" ? So it is consistent with the others.

Reply:
Sure